src/main/java/uk/gov/justice/digital/delius/transformers/CustodialStatusTransformer.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.digital.delius.transformers;
+
+import org.springframework.stereotype.Component;
+import uk.gov.justice.digital.delius.data.api.CustodialStatus;
+import uk.gov.justice.digital.delius.data.api.KeyValue;
+import uk.gov.justice.digital.delius.jpa.standard.entity.Disposal;
+import uk.gov.justice.digital.delius.jpa.standard.entity.Release;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Component
+public class CustodialStatusTransformer {
```

Comment:
I think Andy recently switched most of the transformers to be static methods where it makes sense.  Mainly because we don't want to be mocking transformers anyway so there's no real value in making them beans.